apps/etterlatte-utbetaling/src/main/kotlin/no/nav/etterlatte/utbetaling/iverksetting/utbetaling/UtbetalingMapper.kt

```diff
@@ -88,6 +85,23 @@ class UtbetalingMapper(
             )
         }
 
+    private fun klassifikasjonskode(
+        saktype: Saktype,
+        utbetalingsperiode: Utbetalingsperiode,
+    ) = when (saktype) {
+        Saktype.OMSTILLINGSSTOENAD -> {
+            OppdragKlassifikasjonskode.OMSTILLINGSTOENAD_OPTP
+        }
+        Saktype.BARNEPENSJON -> {
+            // TODO i en overgangsperiode vil det finnes en del vedtak uten regelverk satt - kompenserer for dette her
```

Comment:
Hva er egentlig todo-en her? Å fjerne dette etter en viss tid... eller?

Reply:
Vil det finnes i en overgangsperiode? Skjer ikke denne mappingen når vi attesterer vedtak, og da burde de allerede ha det med? Eller mapper vi fra databasen og trenger å håndtere historisk manglende felter -- og da har vi nok en lengre periode mtp. måten vi henter ut saker for konsistensavstemming eventuelt?

Reply:
Regelverk-feltet settes når vi oppretter / oppdaterer et vedtak. Dvs at når et vedtak fattes, så settes dette feltet basert på hva som ligger i beregning. Utfordringen her oppstår ved at det allerede ligger inne en del vedtak i prod som er fattet, men ikke attestert. Disse vil eksempelvis ikke få regelverk satt før det skippes til utbetaling. Tilsvarende for vedtak hvor beregning allerede er gjort. Disse har heller ikke fått satt regelverk i vedtaket. Målet er ikke å fikse alle disse vedtakene bakover i tid, for de blir riktig videre når de revurderes / reguleres. Men tar gjerne en liten prat på det imorgen for å sjekke at det ikke er andre måter det kan løses mer elegant.

Reply:
ah, da skjønner jeg. tenker den gir mening, men sikkert ikke dumt å lage en jira-oppgave og evt. tenke om vi har en god måte å sjekke om det er "ok" å fjerne mappingen her (vi har vel ikke noen eksplisitt limit på hvor lenge en sak kan ligge til attestering)

Reply:
Godt poeng, opprettet en sak her for oppfølging: https://jira.adeo.no/browse/EY-4564